amqp_connection.c

```diff
@@ -85,17 +85,17 @@ zend_object_handlers amqp_connection_object_handlers;
 
 static int php_amqp_connection_resource_deleter(PHP5to7_zend_resource_le_t *el, amqp_connection_resource *connection_resource TSRMLS_DC)
 {
-    if (Z_RES_P(el)->ptr == connection_resource) {
+	if (Z_RES_P(el)->ptr == connection_resource) {
 		return ZEND_HASH_APPLY_REMOVE | ZEND_HASH_APPLY_STOP;
-    }
+	}
 
 	return ZEND_HASH_APPLY_KEEP;
 }
 
 static PHP5to7_param_str_len_type_t php_amqp_get_connection_hash(amqp_connection_params *params, char **hash) {
 	return spprintf(hash,
 					0,
-					"amqp_conn_res_h:%s_p:%d_v:%s_l:%s_p:%s_f:%d_c:%d_h:%d_cacert:%s_cert:%s_key:%s",
+					"amqp_conn_res_h:%s_p:%d_v:%s_l:%s_p:%s_f:%d_c:%d_h:%d_cacert:%s_cert:%s_key:%s:%d",
```

Comment:
Should be `…sasl_method:%d` for consistency

Reply:
@lstrojny 
addressed